roles/generate-jenkins/templates/README_SNIPPETS/UPDATING_INFO.j2

```diff
@@ -0,0 +1,69 @@
+## Updating Info
+
+Most of our images are static, versioned, and require an image update and container recreation to update the app inside. With some exceptions (ie. nextcloud, plex), we do not recommend or support updating apps inside the container. Please consult the [Application Setup](#application-setup) section above to see if it is recommended for the image.
+
+Below are the instructions for updating containers:
+
+### Via Docker Compose
+
+* Update images:
+    * All images:
+        ```bash
+        docker-compose pull
+        ```
+    * Single image:
+        ```bash
+        docker-compose pull {{ project_name }}
+        ```
+* Update containers:
+    * All containers:
+        ```bash
+        docker-compose up -d
+        ```
+    * Single container:
+        ```bash
+        docker-compose up -d {{ project_name }}
+        ```
+* You can also remove the old dangling images:
+    ```bash
+    docker image prune
+    ```
+
+### Via Docker Run
+
+* Update the image:
+    ```bash
+    docker pull lscr.io/{{ lsio_project_name_short }}/{{ project_name }}:{{ release_tag }}
+    ```
+* Stop the running container:
+    ```bash
+    docker stop {{ project_name }}
+    ```
+* Delete the container:
+    ```bash
+    docker rm {{ project_name }}
+    ```
+* Recreate a new container with the same docker run parameters as instructed above (if mapped correctly to a host folder, your `/config` folder and settings will be preserved)
+* You can also remove the old dangling images:
+    ```bash
+    docker image prune
+    ```
+
+### Via Watchtower auto-updater (only use if you don't remember the original parameters)
+
+* Pull the latest image at its tag and replace it with the same env variables in one run:
+
+    ```bash
+    docker run --rm \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      containrrr/watchtower \
+      --run-once {{ project_name }}
+    ```
+
+* You can also remove the old dangling images: `docker image prune`
+
+**Note:** We do not endorse the use of Watchtower as a solution to automated updates of existing Docker containers. In fact we generally discourage automated updates. However, this is a useful tool for one-time manual updates of containers where you have forgotten the original parameters. In the long term, we highly recommend using [Docker Compose]({{ lsio_docs_url }}/general/docker-compose).
+
+### Image Update Notifications - Diun (Docker Image Update Notifier)
+
+* We recommend [Diun](https://crazymax.dev/diun/) for update notifications. Other tools that automatically update containers unattended are not recommended or supported.
```

Comment:
This could also be a admonition

Reply:
Fixed in https://github.com/linuxserver/docker-jenkins-builder/pull/217/commits/22f544cc406c43c6641ccdc4c32fa84a16ad7f30